WordPress/Classes/Utility/Migration/ContentMigrationCoordinator.swift

```diff
@@ -34,7 +45,7 @@
         ensureBackupDataDeletedOnLogout()
     }
 
-    enum ContentMigrationCoordinatorError: LocalizedError {
+    enum Errors: String, LocalizedError {
```

Comment:
Non-blocking nit / preference: 

A singular `Error` seems like a better fit for this, but could introduce extra confusion with [the protocol name](https://developer.apple.com/documentation/swift/error). `MigrationError` may be a good fit for its purpose and reduces the nested type stuttering (e.g. `ContentMigrationCoordinator.ContentMigrationCoordinatorError`), but that's just my opinion. 😄

Reply:
I don't have any preference, but I definitely don't want a Java-esque `ContentMigrationCoordinator.ContentMigrationCoordinatorError` 😂 . Anywho, addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/19787/commits/24e403aba234a86ebee1e96343d8727b9155cc78.